src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -14,11 +14,52 @@
 SKU_TIER_MAP = {'Basic': 'b', 'GeneralPurpose': 'gp', 'MemoryOptimized': 'mo'}
 
 
+def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_wait=False,
+                   location=None, administrator_login=None, administrator_login_password=None, backup_retention=None,
+                   geo_redundant_backup=None, ssl_enforcement=None, storage_size=None, tags=None, version=None):
+    provider = 'Microsoft.DBForMySQL' if isinstance(client, ServersOperations) else 'Microsoft.DBforPostgreSQL'
+    parameters = None
+    if provider == 'Microsoft.DBForMySQL':
+        import azure.mgmt.rdbms.mysql.models
+        parameters = azure.mgmt.rdbms.mysql.models.ServerForCreate()
+        parameters.sku = azure.mgmt.rdbms.mysql.models.Sku()
+        parameters.properties = azure.mgmt.rdbms.mysql.models.ServerPropertiesForDefaultCreate()
+        parameters.properties.storage_profile = azure.mgmt.rdbms.mysql.models.StorageProfile()
+    elif (provider == 'Microsoft.DBforPostgreSQL'):
+        import azure.mgmt.rdbms.postgresql.models
+        parameters = azure.mgmt.rdbms.postgresql.models.ServerForCreate()
+        parameters.sku = azure.mgmt.rdbms.postgresql.models.Sku()
+        parameters.properties = azure.mgmt.rdbms.postgresql.models.ServerPropertiesForDefaultCreate()
+        parameters.properties.storage_profile = azure.mgmt.rdbms.postgresql.models.StorageProfile()
+
+    parameters.location = location
+    parameters.tags = tags
+    parameters.sku.name = sku_name
+    parameters.properties.version = version
+    parameters.properties.ssl_enforcement = ssl_enforcement
+    parameters.properties.storage_profile.backup_retention_days = backup_retention
+    parameters.properties.storage_profile.geo_redundant_backup = geo_redundant_backup
+    parameters.properties.storage_profile.storage_mb = storage_size
+    parameters.properties.administrator_login = administrator_login
+    parameters.properties.administrator_login_password = administrator_login_password
```

Comment:
This could be a lot more concise:
```Python
import A, B, C
parameters = A(
  prop1=prop1,
  b=B(
    foo=blah,
    bar=dah
  ),
  c=C(
    prop_a=propa,
    prop_b=propb
  )
)
```

This is the way most creates are written.

Reply:
@tjprescott, These objects are nested. Do you want me to follow the same pattern with nesting?

Reply:
Yeah, the nesting is no problem.  It's just more readable to initialize all the objects in-line rather than creating them empty and filling them up after-the-fact